src/js/jsx/tools/SuperselectOverlay.jsx

```diff
@@ -400,6 +402,53 @@ define(function (require, exports, module) {
         },
 
         /**
+         * Checks to see if the given position is under
+         * any always_propagate policies, where mouse events should go to PS
+         * and we shouldn't highlight layers
+         *
+         * We reverse iterate through policies to find the top most one 
+         * that should be accounted for at the location
+         *
+         * @param {number} x
+         * @param {number} y
+         *
+         * @return {boolean} True if the pointer is under an always propagate policy
+         */
+        _positionUnderAlwaysPropagatePolicy: function (x, y) {
+            var policyStore = this.getFlux().store("policy"),
+                modifierStore = this.getFlux().store("modifier"),
+                modifiers = modifierStore.getState(),
+                modifierBits = keyUtil.modifiersToBits(modifiers),
+                pointerPolicies = policyStore.getMasterPointerPolicyList(),
+                underAlways = false;
+
+            _.forEachRight(pointerPolicies, function (policy) {
+                if (!policy.area || policy.modifiers !== modifierBits) {
+                    return;
+                }
+
+                var area = {
+                        left: policy.area[0],
+                        top: policy.area[1],
+                        right: policy.area[0] + policy.area[2],
+                        bottom: policy.area[1] + policy.area[3]
+                    },
+                    intersects = area.left < x && area.right > x &&
+                        area.top < y && area.bottom > y;
+
+                if (intersects) {
+                    if (policy.action === UI.policyAction.ALWAYS_PROPAGATE) {
+                        underAlways = true;
+                    } else {
+                        underAlways = false;
+                    }
+                }
+            }.bind(this));
```

Comment:
We always prefer the receiver argument of explicit binds when available, as it is here.